### ISSUE TITLE
Reset on PG7 reboot

### ISSUE THREAD

**Issue by AlexKlimaj** [7Tunnels/pg7-msp430#12]
The guys want the PG7 when one types "reboot" on the linux command line.  As the PG7 is now the reboot cycle stops after the power goes off.

Any ideas on what could be causing this?   I can see the evaluation board does reset (most of the time) when you type reboot at the command line.

**Comment by AlexKlimaj** [comment]
I've figured out what is going on with the reboot reset issue.

It is resetting because the IMX6_ONOFF_PIN pin we use to signal to the MSP430 is going low and then the MSP430 is shutting off the PMIC.

I've added a timer to wait X seconds, check the IMX6_ONOFF_PIN pin again, if it is low, shut off the pmic. If it is not low, don't shut off the PMIC because this means it was a reboot not shutdown.

However, it appears that the IMX6 is not driving IMX6_ONOFF_PIN high upon reboot. I thought we had this set in uboot to drive high first thing on boot? Why wouldn't it drive high on a reboot?
